Source/Core/DiscIO/NFSBlob.cpp

```diff
@@ -167,9 +164,9 @@ NFSFileReader::NFSFileReader(std::vector<NFSLBARange> lba_ranges, std::vector<Fi
 
 std::unique_ptr<BlobReader> NFSFileReader::CopyReader() const
 {
-  std::vector<File::IOFile> new_files{};
-  for (const File::IOFile& file : m_files)
-    new_files.push_back(file.Duplicate("rb"));
+  std::vector<File::DirectIOFile> new_files{};
+  for (const auto& file : m_files)
+    new_files.push_back(file);
```

Comment:
Now that you don't need to call `Duplicate`, you should be able to use the vector copy constructor instead of writing a loop and using `std::move`.